### PR TITLE
This PR Is A Test  (Please don't review)

### PR DESCRIPTION
**Pull request by FioraAeterna**
This is a big PR with all my current changes aimed at speeding up RS3 and friends. It isn't intended to be pushed, but rather I'm creating it so that there's convenient buildbot builds for it. This is useful since testing all my changes at once is more likely to catch bugs.

### CODE REVIEW

**Comment by waddlesplash** [comment]
>  PR cannot be merged, please rebase.

**Comment by degasus** [comment]
LGTM

File: Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp [hunk 1]
```diff
@@ -381,43 +402,79 @@ void Jit64::DoMergedBranch()
...
+		else
+			destination = nextPC + SignExt16(next.BD << 2);
+		jumpInBlock = destination > nextPC && destination < js.blockEnd;
+	}
+	bool condition = !!(next.BO & BO_BRANCH_IF_TRUE) ^ jumpInBlock;
 
 	gpr.UnlockAll();
 	gpr.UnlockAllX();
-	FixupBranch pDontBranch;
+	FixupBranch pBranch;
```
Comment (Armada651) [inline Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp]: Dolphin does not use Hungarian notation prefixes.

**Comment by kkkllleee** [comment]
Your suggestion passed all the checks, you should push it to be merged.

**Comment by BhaaLseN** [comment]
Can you please stop this, @kkkllleee? You don't need to ping each and every PR (especially ones that are never meant to be reviewed or merged, such as this current one).

**Comment by kkkllleee** [comment]
@BhaaLseN But why not? It works. Is not like Fiora couldn't change his mind

**Comment by JosJuice** [comment]
Sure, it "works"... except it crashes in several levels in Star Wars: Rouge Leader, and it could potentially have more issues that nobody has discovered yet. When Fiora says it shouldn't be pushed, she has a reason for that.

**Comment by phire** [comment]
I believe it also breaks audio in some games.

**Comment by kkkllleee** [comment]
@JosJuice  Ok, it's not like I'm forcing anything.

**Comment by MoochMcGee** [comment]
@kkkllleee Protip: Fiora is a girl.

**Comment by kkkllleee** [comment]
@MoochMcgee I corrected it.

**Comment by kkkllleee** [comment]
@BhaaLseN By the way, those times I was new and I really didn't know what to say, but now I promised it wouldn't happen again, it's just that in this instance I feeleld like it was fitting, since it just seemed rare that it passed all the tests and yet Fiora didn't want to merge.

**Comment by waddlesplash** [comment]
> I felled like it was fitting, since it just seemed rare that it passed all the tests and yet Fiora didn't want to merge.

No. Unless you understand *every single line* of code in the PR, then you don't suggest it should be merged. Also, "all the tests" consists of making sure the code compiles, as a FifoCI run was not triggered on this PR.

**Comment by JosJuice** [comment]
A hint: The tests are pretty much irrelevant to whether a PR will be merged or not. If one of the checks is failing, the submitter is able to find that out easily on their own, so there's generally no need to comment about it. Automated tests are only able to do some very basic checks, so there is always a need for humans to do the majority of the code review process, meaning that a PR could be terrible even if all checks are passing.

**Comment by kkkllleee** [comment]
@JosJuice I see, is there any way I can get educated in how dolphin works in detail?

**Comment by comex** [comment]
Aanyway.  It's been 3.5 months so make a new one if you still want this.

**Pull request closed without merging**